numba/tests/test_typedlist.py

```diff
@@ -530,6 +533,60 @@ def test_repr(self):
         expected = "ListType[int32]([1, 2, 3])"
         self.assertEqual(expected, repr(l))
 
+    def test_repr_long_list_ipython(self):
+        # Test repr of long typed Lists in an IPython session
+        base_cmd = [sys.executable, '-m', 'IPython']
+        base_cmd += ['--quiet', '--quick', '--no-banner', '--colors=NoColor']
+        try:
+            ver = subprocess.check_output(base_cmd + ['--version'])
+        except subprocess.CalledProcessError as e:
+            self.skipTest("ipython not available: return code %d"
+                          % e.returncode)
+        ver = ver.strip().decode()
+        print("ipython version:", ver)
```

Comment:
Please delete this, it adds noise to test output.

Reply:
Thanks for your review. Just to clarify, you mean lines 545-546, right?

Reply:
I think @stuartarchibald means the `print` statement.

Reply:
@esc Thanks, refactored the unit test and removed the `print` statement.